specification/oars-1.1.md

```diff
@@ -3,7 +3,7 @@ OARS 1.0 and 1.1 specification
 
 This specification gives the semantics for the OARS 1.0–1.1 XML format. The
 structure of the format is specified in the
-[accompanying XML schema](./oars-1.1.rng).
```

Comment:
:joy: :game_die:

Reply:
It’s because I originally started writing the schema as RelaxNG XML (whose file suffix is traditionally `.rng`), before I came back to my senses and did it in RelaxNG Compact (`.rnc`).